src/commands/build.ts

```diff
@@ -85,36 +83,58 @@ export class BuildCommand implements Command {
     },
   ];
 
-  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
+  async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
-    const build = require('../build/build').build;
+    let build = require('../build/build').build;
+    const hasCLIArgumentsPassed = this.args.some(arg => {
+      return typeof options[arg.name] !== 'undefined';
```

Comment:
I agree this falls closer to paranoia than reasonably common. But to me its the exact same argument as using semi-colons: "it's almost always fine, but why add risk unnecessarily?"

Happy to bring this up in a weekly standup if this is something you'd like us to come together on as a team.